vmware_aria_operations_integration_sdk/adapter_configurations/adapter_templates/java/sample_adapter/src/main/java/Adapter.java.template

```diff
@@ -1,100 +1,326 @@
+        }
+        finally {
+            logger.debug("Returning collection result" +  result.getJson());
+            timer.stop();
         }
-        return map;
+
+        return result;
     }
 
-    private static String test(JsonNode adapterDefinition) {
-        ObjectMapper mapper = new ObjectMapper();
-        ObjectNode rootNode = mapper.createObjectNode();
-
-        if (adapterDefinition.path("adapter_key")
-                .path("identifiers")
-                .get(0)
-                .path("value")
-                .asText()
-                .equals("bad")){
-            rootNode.put("errorMessage", "Oh no something went horribly wrong!");
+    public EndpointResult getEndpoints(AdapterInstance adapterInstance) {
+        Logger logger = AdapterLogger.getLogger();
+        Timer timer = new Timer("Get Endpoints");
+        EndpointResult result = new EndpointResult();
+        try {
+            // In the case that an SSL Certificate is needed to communicate to the target,
+            // add each URL that the adapter uses here. Often this will be derived from a
+            // 'host' parameter in the adapter instance. In this Adapter we don't use any
+            // HTTPS connections, so we won't add any. If we did, we might do something like
+            // this:
+            // result.with_endpoint(adapter_instance.get_identifier_value("host"))
+            //
+            // Multiple endpoints can be returned, like this:
+            // result.with_endpoint(adapter_instance.get_identifier_value("primary_host"))
+            // result.with_endpoint(adapter_instance.get_identifier_value("secondary_host"))
+            //
+            // This 'get_endpoints' method will be run before the 'test' method,
+            // and VMware Aria Operations will use the results to extract a certificate from
+            // each URL. If the certificate is not trusted by the VMware Aria Operations
+            // Trust Store, the user will be prompted to either accept or reject the
+            // certificate. If it is accepted, the certificate will be added to the
+            // AdapterInstance object that is passed to the 'test' and 'collect' methods.
+            // Any certificate that is encountered in those methods should then be validated
+            // against the certificate(s) in the AdapterInstance.
+
+            // TODO: Add any additional endpoints if any
+        } catch (Exception e){
+            logger.info(e.getMessage());
         }
+        finally {
+            logger.debug("Returning endpoints: " + result.getJson());
+            timer.stop();
+        }
+        return result;
+    }
+
+    // Main entry point of the adapter. You should not need to modify anything below this line.
+    public static void main(String[] args) throws FileNotFoundException {
+        // For log rotation and other logging setups, consider using a Java logging library or framework.
```

Comment:
Both of the templates have this comment in the main function -- is this a holdover from before `AdapterLobber.setupLogging` was implemented?